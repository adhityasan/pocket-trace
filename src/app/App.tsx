import ErrorBoundary from "@components/ErrorBoundary";
import RootNavigation from "@navigations/root";

export default function App() {
  return (
    <ErrorBoundary>
      <RootNavigation />
    </ErrorBoundary>
  );
}
