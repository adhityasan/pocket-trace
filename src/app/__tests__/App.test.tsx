import { create, act } from "react-test-renderer";

import App from "../App";

describe("App", () => {
  it("should match snapshot", () => {
    const result = create(<App />).toJSON();
    act(() => {
      expect(result).toMatchSnapshot();
    });
  });
});
