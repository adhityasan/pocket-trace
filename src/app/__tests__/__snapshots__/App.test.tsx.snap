// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNSScreenStack
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      collapsable={false}
      forwardedRef={[Function]}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onTransitionProgress={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      stackPresentation="push"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backgroundColor="rgb(255, 255, 255)"
        color="rgb(0, 122, 255)"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={true}
        hideBackButton={false}
        largeTitleHideShadow={false}
        title="Home"
        titleColor="rgb(28, 28, 30)"
        topInsetEnabled={false}
        translucent={false}
      />
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            Object {
              "backgroundColor": "rgb(242, 242, 242)",
            },
            undefined,
          ]
        }
      >
        <RCTSafeAreaView
          emulateUnlessSupported={true}
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              },
              Object {
                "backgroundColor": "#232323",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "paddingHorizontal": 5,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "fontSize": 30,
                    "fontWeight": "bold",
                    "textAlign": "center",
                  },
                  Object {
                    "color": "#919ff0",
                  },
                ]
              }
            >
              Home Screen
            </Text>
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#eee",
                    "textAlign": "center",
                  }
                }
              >
                see about screen ?
              </Text>
            </View>
          </View>
        </RCTSafeAreaView>
      </View>
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
